das-cli/src/commands/config/config_cli.py

```
@@ -73,9 +73,21 @@ def __init__(
 
     def _set_file_path(self, save_path) -> None:
         self._settings.set_path(save_path)
-
         self._settings.rewind()
 
+        load_error = getattr(self._settings._store, "_load_error", None)
```

Pylint (4.0.6)

[warning] Access to a protected member _store of a client class

(W0212)